### PR TITLE
client: Fix the issue that TiKV reconnecting may make CDC stuck for a while

### PR DESCRIPTION
**Pull request by MyonKeminta**
### What problem does this PR solve? 

This PR tries to fix the issue about reconnecting TiKV may make CDC stuck for a while, and added some debug logs for investigation.

### Check List 

Tests 

 - Unit test
- Integration test

### Release note

- Fix the issue that reconnecting to TiKV may cause CDC stuck for a while

### CODE REVIEW

File: cdc/kv/client_test.go [hunk 1]
```diff
@@ -61,6 +70,128 @@ func (s *clientSuite) TestUpdateCheckpointTS(c *check.C) {
...
+	defer func() {
+		if ch2 != nil {
+			close(ch2)
+		}
+	}()
+	server2 := newMockService(c, 23376, ch2)
+	defer func() {
+		// if server1 != nil {
+		// 	server1.Stop()
+		// }
```
Comment (overvenus) [inline cdc/kv/client_test.go:135]: Could you remove these comments?

File: cdc/kv/client_test.go [hunk 2]
```diff
@@ -61,6 +70,128 @@ func (s *clientSuite) TestUpdateCheckpointTS(c *check.C) {
...
+	cluster.AddStore(2, "localhost:23376")
+	cluster.Bootstrap(3, []uint64{1, 2}, []uint64{4, 5}, 4)
+
+	cdcClient, err := NewCDCClient(pdClient, kvStorage.(tikv.Storage))
+	c.Assert(err, check.IsNil)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	eventCh := make(chan *model.RegionFeedEvent, 10)
+	go func() {
+		err := cdcClient.EventFeed(ctx, util.Span{Start: []byte("a"), End: []byte("b")}, 1, eventCh)
```
Comment (amyangfei) [inline cdc/kv/client_test.go:157]: `cdcClient.EventFeed` may not return before the test case tear down, what about adding some wait mechanism

File: cdc/kv/client_test.go (hunk 1 above)
Comment (MyonKeminta) [reply cdc/kv/client_test.go:135]: Oh, sorry.

**Comment by leoppro** [comment]
LGTM

**Pull request merged**